_src/recipes/010-add-a-sitemap.md

```diff
@@ -16,20 +26,20 @@ eleventyExcludeFromCollections: true
 ---
 <?xml version="1.0" encoding="utf-8"?>
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
-    {% for page in collections.all %}
+    {% raw %}{% for page in collections.all %}
         <url>
             <loc>{{ site.url }}{{ page.url | url }}</loc>
             <lastmod>{{ page.date.toISOString() }}</lastmod>
             <changefreq>{{page.data.changeFreq}}</changefreq>
```

Comment:
Where does `page.data.changeFreq` come from? Is that supplied by Eleventy or does it need to be included in the front matter? Is this value optional?